.github/workflows/behave_next_python.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: Run behave with unsupported Python versions
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - main
+      - develop
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  behave_next_python:
+    name: Run Behave
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: ['3.13']
+
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v3
+
+      - name: Install Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y docker-compose make
+
+      - name: Check Docker state (pre-build)
+        run: docker ps
+
+      - name: Build Docker images
+        run: make build
+        env:
+          PYTHON_IMAGE_VER: "${{ matrix.python-version }}"
+
+      - name: Migrate Database
+        run: |
+          make migrate || echo "::warning:: migrate failed on future Python version ${{ matrix.python-version }}."
```

Comment:
why the echoes? since this is basically the same workflow as the not future one, why are they in here but not the other workflow?

Reply:
The echoes aren't the important part, it's the `||`, which makes this step not fail (thus allowing this entire workflow to fail, but not marking it as a "failure").

Reply:
I get that I guess I just am confused why we want them here, but not in the other workflow as well for the same reason?